examples/chardev.c

```diff
@@ -27,10 +27,16 @@ static ssize_t device_write(struct file *, const char __user *, size_t,
 /* Global variables are declared as static, so are global within the file. */
 
 static int major; /* major number assigned to our device driver */
+
 /* Is device open? Used to prevent multiple access to device */
 static atomic_t already_open = ATOMIC_INIT(0);
+
+enum {
```

Comment:
Don't do that. Each constant in this `enum` should have a common prefix. Let's say, `CDEV_`. Also, shorten the names of constants.